programmers/k번째수.js

```diff
@@ -0,0 +1,11 @@
+function solution(array, commands) {
+    const result = [];
+    
+    for (let i = 0; i < commands.length; i++) {
+       const arr = array.slice(commands[i][0] - 1, commands[i][1])
+                        .sort((a, b) => a - b);
+       result.push(arr[commands[i][2] - 1]);
+    };
```

Comment:
```suggestion
  commands.forEach((command) => {
        result.push(array.slice(command[0] - 1, command[1]).sort((a, b) => a - b)[command[2] - 1]);
    });
```
forEach()라는 메소드를 사용하시면 배열의 원소 각각을 처리할 수 있답니다. 반복되는 범위를 지정해주어야 하는 for과는 달리 forEach()는 배열의 원소 만큼 실행하고, 처리하기에 용이하고, index등을 이용할 수 있어 연습하시면 엄청 편리해요! https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach